spec/index.bs

```diff
@@ -3634,6 +3637,23 @@ All [=depth-or-stencil formats=] are renderable.
     1. Return `null`.
 </div>
 
+Use of some texture formats require a feature to be enabled on the {{GPUDevice}}. Because support
+for these formats may not be implemented on all systems, the associated enums may not be present.
+In order to normalize behavior across implementations attempting to use a format that requires a
+feature when the associated feature is not enabled on the device will throw an exception, making the
+behavior the same as if an undefined enum was used.
```

Comment:
tweaks for clarity:
```suggestion
Use of some texture formats require a feature to be enabled on the {{GPUDevice}}. Because new
formats can be added to the specification, those enum values may not be known by the implementation.
In order to normalize behavior across implementations, attempting to use a format that requires a
feature will throw an exception if the associated feature is not enabled on the device. This makes
the behavior the same as the format is unknown to the implementation.
```

Reply:
Nice edit, thanks! I rolled this into the commit addressing the rest of your feedback.